src/cli/domain/get-mocked-plugins.js

```diff
@@ -53,23 +54,47 @@ var registerDynamicMocks = function(mocks, logger){
   }).filter(function(p){ return p; });
 };
 
-module.exports = function(logger){
+var findPath = function(pathToResolve, fileName) {
+
+  var rootDir = fs.realpathSync('.'),
+      fileToResolve = path.join(pathToResolve, fileName);
+
+  if (!fs.existsSync(fileToResolve)) {
+    if (pathToResolve === rootDir) {
+      return undefined;
+    } else {
+      var getParent = function(x){ return x.split('/').slice(0, -1).join('/'); },
+          parentDir = pathToResolve ? getParent(pathToResolve) : rootDir;
+
+      return findPath(parentDir, fileName);
+    }
+  }
+  
+  return fileToResolve;
+};
+
+module.exports = function(logger, componentsDir){
+  componentsDir = path.resolve(componentsDir || '.');
+
   var plugins = [],
-      ocJsonPath = path.resolve(settings.configFile.src);
+      ocJsonFileName = settings.configFile.src.replace('./', ''),
+      ocJsonPath = findPath(componentsDir, ocJsonFileName);
 
-  if(!fs.existsSync(ocJsonPath)){
+  if(!ocJsonPath){
     return plugins;
   }
 
-  var content = fs.readJsonSync(ocJsonPath);
+  var content = fs.readJsonSync(ocJsonPath),
+      ocJsonLocation = ocJsonPath.slice(0, -ocJsonFileName.length);
+
   if(!content.mocks || !content.mocks.plugins){
     return plugins;
   }
 
   logger.log(colors.yellow(strings.messages.cli.REGISTERING_MOCKED_PLUGINS));
 
   plugins = plugins.concat(registerStaticMocks(content.mocks.plugins.static, logger));
-  plugins = plugins.concat(registerDynamicMocks(content.mocks.plugins.dynamic, logger));
+  plugins = plugins.concat(registerDynamicMocks(content.mocks.plugins.dynamic, ocJsonLocation, logger));
```

Comment:
I am also thinking about preserve the same "pattern" between the methods (registerStaticMocks and registerDynamicMocks) such as adding the extra parameter either at the very beginning or after "logger" but not in between 😅 

`registerDynamicMocks(ocJsonLocation, content.mocks.plugins.dynamic, logger)`

or

`registerDynamicMocks(content.mocks.plugins.dynamic, logger, ocJsonLocation)`